rotkehlchen/exchanges/coinbase.py

```diff
@@ -693,7 +703,10 @@ def query_online_deposits_withdrawals(
                 if 'type' not in tx:
                     continue
                 if tx['type'] == 'send':
-                    raw_data.append(tx)
+                    if 'to' in tx:
+                        raw_data.append(tx)
+                    if 'from' in tx and 'network' in tx and 'hash' in tx['network']:
+                        raw_data.append(tx)
```

Comment:
This could be a little simpler:
```suggestion
                    if (
                        'to' in tx or
                        (all (k in tx for k in ('from', 'network')) and 'hash' in tx['network'])
                    ):
                        raw_data.append(tx)
```